README.md

```diff
@@ -49,6 +49,12 @@ To run a much bigger demo, run the command:
 python main.py data/huge.csv 400
 ```
 
+To run a very small demo, run the command:
+
+```bash
+python main.py data/very_small.csv 10
```

Comment:
I like the additional example, we might as well add it to the tests as well.

Reply:
Agreed would be good to test.  The testing strategy is not very clear to me though.  Currently, this repo tests the the small.csv example with the hybrid solver and checks the numerical results, but I'd be concerned about checking against results from a probabilistic solver, since it may not necessarily pass.  I noticed that the tests in some of the other example repos (e.g., image-segmentation) don't check the numerical results of the solver.

Is there some kind of best practice for testing with these examples?  I'd be more inclined to verify results using the ExactSolver where possible (in this case could be done for small.csv and very_small.csv), but that would require some modification to `main.py`.  Other unit tests that might be helpful could include verifying that certain inputs to the BQM produce the expected energy (e.g., on the larger problems where we can't run the exact solver).

Reply:
I imagine that some fiddling with [patch](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) would let you patch in the ExactSolver as a substiture for the `LeapHybridSampler`.

Reply:
I'd be inclined to add a command argument switch to change to the exact solver or even refactor the code to be more modular so that individual pieces can be tested as opposed to testing it as a script.

Reply:
I agree with making it more modular and testing smaller pieces with ExactSolver. n-queens also does this.

Reply:
I'm fine with any of these code approaches, but I like the overall idea of using ExactSolver() for the very small problems. An advantage of this is that it would become non-probabilistic and the tests would work again. The "small" case was designed for that. I can't remember if the "50 test" is still alive, but it could come back, with ExactSolver.